src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#moyo_logo"
+        class="moyo__logo"
+      >
```

Comment:
The `href` attribute of the logo link should point to the home page or the top of the page, typically `href="#"` or `href="index.html"`, instead of `#moyo_logo` which doesn't correspond to a valid section.